src/services/identity/IIdentity.ts

```diff
@@ -0,0 +1,243 @@
+import * as dotenv from 'dotenv';
```

Comment:
This file is no longer on develop, so it seems that this getting re-introduced is because of a bad merge.

Reply:
It is not a bad merge.

Currently, in develop branch we don't have `src/services/identity/IIdenity.ts` because this file context will be moved in `src/services/identity/IIdentity.ts`.

1. I created this file for fixing problem with cycle import.
2. I refactored code in `IIdentity`.
  -  we have identical same methods: 6 methods + a lot of not implemented methods in `Local` and `Unauthorized`.
    - `resolveDid` (return `DIDDocument` = `DID Resolution`), resolve (support query parameters = `Dereferencing`), `verifyCredential`, `verifyPresentation`, `checkStatusList2021`, `searchStatusList2021` - same methods in different subclasses such as:
    - `Unauthorized`, `Postgres`, `Local`.
    For that reason I decided to refactor code. In currently design looks like that:
    1. `IIdentity` (interface) - provide all methods which should be implemented.
    2. `AbstractIdentity` (abstract class) - which implement all `IIdentity` methods like `not impelemented`.
    3. `DefaultIdentity` (class) - which extend `AbstractIdentity` and implement only 6 base methods which will be same for subclasses.
    4. `Postgres`, `Local`, `Unauthorized` (class) - which extend `DefaultIdentity` and implement specific methods which will be supported for these classes.